lib/publiccloud/instance.pm

```diff
@@ -66,7 +66,7 @@ sub scp {
     $from =~ s/^remote:/$url/;
     $to   =~ s/^remote:/$url/;
 
-    my $ssh_cmd = sprintf('scp %s -i "%s" "%s" "%s"',
+    my $ssh_cmd = sprintf('scp %s -i "%s" %s %s',
```

Comment:
I don't get it, why do we need to remove these quotes? What about a file name with spaces, I would avoid let the caller handle the escaping.